### ISSUE TITLE
Exceptions doesn't log properties set in addTelemetryInitializer

### ISSUE THREAD

**Issue by riteshbhoi** [microsoft/ApplicationInsights-JS#954]
We are using @microsoft/applicationinsights-web version  2.0.0-beta.3 in our react application. We initialize few custom dimension and attach it to base telemetry context using addTelemetryInitializer method. So all the telemetry types (events, trace, dependencies) logs them except exceptions. Is it a bug in code which is causing this issue?

**Comment by markwolff** [comment]
Hi @riteshbhoi can you try using the latest version? Also if that does not work, can you provide a repro or code snippet that I can look at?

**Comment by tbolon** [comment]
Hi @markwolff, I detected the same problem using the latest CDN version.

But in my case, it's the PageView event which is missing the additional properties.

Here is my code:

̀```
    aisdk.queue.push(() => {
        appInsights.addTelemetryInitializer(envelope => {
            envelope.data["Environment"] = "dev";
        });
    });

    aisdk.trackTrace({message: 'Local trace', severityLevel: 'Information' } );
</script>
```

The Trace event includes the custom property, but not the PageView event.

**Comment by markwolff** [comment]
@tbolon Released the fix with 2.2.0
